dev/ComboBox/ComboBox_themeresources.xaml

```diff
@@ -704,7 +701,7 @@
                             Background="Transparent"
                             Margin="4,4,4,4"
                             Visibility="Collapsed"
-                            contract7Present:CornerRadius="{StaticResource ComboBoxDropDownButtonBackgroundCornerRadius}"
```

Comment:
Is it expected that we want to change the corner-radii for the different parts? Templatebinding the corner radius on the template parts would make them have the same value while they currently have different values.

Reply:
It seems I didn't realize what that button was. I looked into the other text controls and they do use TemplateBinding for that, so I'll fix it.

Reply:
Nevermind, that's what I just did. _(it's a bit late here)_
But it seems to be the case. Buttons in controls like PasswordBox, AutoSuggestBox and NumberBox take the corner radius of the box, otherwise they simply use ControlCornerRadius.